packages/react-error-overlay/package.json

```diff
@@ -37,7 +37,7 @@
     "babel-eslint": "9.0.0",
     "babel-jest": "23.6.0",
     "babel-loader": "8.0.5",
-    "babel-preset-react-app": "^7.0.2",
+    "@fs/babel-preset-react-app": "^1.0.0",
```

Comment:
we don't publish this package right now. and we also probably don't need to update the react-error-overlay with our babel anyways. So I'd prefer to remove this particular change.